### PR TITLE
suthesh/faq-markup-update

### PR DESCRIPTION
**Pull request by suthesan-deriv**
Changes:

-   updated schema on help centre page
-   added schema on affiliate page
-   added schema on payment agent page
-   localize schema contents

## Type of change

-   [x] Bug fix
-   [x] New feature
-   [x] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by sean-binary — COMMENTED** [review]
I think you can remove the changes made to`messages.json` in this PR. I'll extract it in a separate PR, because we need to upload it and download the new `ach.json` keys.

**Review by kevinw-binary — COMMENTED** [review]
We should rebase this PR

File: src/pages/partners/affiliate-ib/index.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
 import React from 'react'
 import styled from 'styled-components'
 import Loadable from '@loadable/component'
+import { Helmet } from 'react-helmet'
 import Hero from './_hero.js'
+import {faq_schema} from './_faq-schema.js'
```
Comment (sean-binary) [inline src/pages/partners/affiliate-ib/index.js:6]:
```suggestion
import { faq_schema } from './_faq-schema.js'
```

File: src/pages/partners/affiliate-ib/index.js [hunk 2]
```diff
@@ -109,6 +111,9 @@ const AffiliateIb = () => {
                     'Join Deriv’s affiliate and IB programmes and get a chance to be a partner with a trusted online trading provider.',
                 )}
             />
+            <Helmet>
+                    <script type="application/ld+json">{JSON.stringify(faq_schema)}</script>
+            </Helmet>
```
Comment (sean-binary) [inline src/pages/partners/affiliate-ib/index.js:116]:
```suggestion
            <Helmet>
                <script type="application/ld+json">{JSON.stringify(faq_schema)}</script>
            </Helmet>
```

File: src/pages/partners/affiliate-ib/index.js (hunk 1 above)
Comment (sean-binary) [reply src/pages/partners/affiliate-ib/index.js:6]: Sorry for the real nitpick man 😂

File: src/pages/partners/payment-agent/index.js [hunk 3]
```diff
@@ -18,6 +20,9 @@ const PaymentAgent = () => {
                     'Know all the details about how you can become the payment agent on Deriv. Send us an email to apply!',
                 )}
             />
+            <Helmet>
+                    <script type="application/ld+json">{JSON.stringify(faq_schema)}</script>
+            </Helmet>
```
Comment (sean-binary) [inline src/pages/partners/payment-agent/index.js:25]:
```suggestion
            <Helmet>
                <script type="application/ld+json">{JSON.stringify(faq_schema)}</script>
            </Helmet>
```

**Review by sean-binary — APPROVED** [review]

**Pull request merged**